frontend/.stylelintrc

```diff
@@ -1,6 +1,12 @@
 {
+	"overrides": [
```

Comment:
Who was asking to install postcss-html and lint styles in html? Please revert this to the original file and remove the postcss-html dependency from package.json